CONTRIBUTING.md

```diff
@@ -155,3 +159,15 @@ export FIRESTORE_EMULATOR_HOST="localhost:<Firebase emulator PORT>"
 - Every pull request should have associated issue(s) on our [issue tracker](https://github.com/Real-Dev-Squad/website-backend/issues).
 - For any non-trivial fixes and features, unit and integration tests must be added. The PR reviewer should not approve/merge PR(s) that lack these.
 - The PR(s) should be merged only after the CI passes.
+
+## Certain issues you may face while running the tests:
+
+- Firestore emulator is not starting
+- Java version is not above 11
+- When we run yarn test, it runs both the unit and integration tests (in this order). So after the unit tests are done, the java process is not killed automatically and when our integration test run it gives error.
+
+## Possible solutions for above issues (in particular order):
+
+- add '--project=< your fire store app name>' flag to both the test scripts. (replace the placeholder)
```

Comment:
you can remove the line "replace the placeholder"
That is self-understandable. Also, It is firestore.  (just a minor change)